charts/sysdig/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sysdig
-version: 1.15.51
+version: 1.15.52
```

Comment:
For `sysdig` chart it's not necessary to bump the version because this is an old helm chart that doesn't include rapid response chart